lib/src/screens/home/home_screen.dart

```diff
@@ -25,6 +29,40 @@ class _HomeScreenState extends State<HomeScreen> {
         () => selectedTab = tab,
       );
 
+  @override
+  void initState() {
+    super.initState();
+
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      final repo = IrmaRepositoryProvider.of(context);
+      final navigator = Navigator.of(context);
+
+      _maybeOpenQrScanner(repo, navigator);
```

Comment:
I suspect that the QR scanner is not automatically opened on start-up now when the app becomes active from background.

Reply:
Yes this does not work anymore. To be honest i thought this was not expected behaviour so omitted it. But i will add it again when reactivating the app, so not just on launch. 👍🏻